Libplanet.Net/Consensus/Context.cs

```diff
@@ -544,6 +746,19 @@ private bool IsValid(Block<T> block)
             }
         }
 
+        /// <summary>
+        /// Creates a <see cref="Vote"/> for <see cref="ConsensusVote"/> or
+        /// <see cref="ConsensusCommit"/>.
```

Comment:
```suggestion
        /// Creates a <see cref="Vote"/> instance for <see cref="ConsensusVote"/> or
        /// <see cref="ConsensusCommit"/> for the <paramref name="round"/> and the <paramref name="hash"/>.
```

Additionally, how about remove `height` param? It seems only `Height` is used.